src/ai/backend/client/cli/scp.py

```diff
@@ -33,14 +33,15 @@ def scp(sess_name, src, dest, port, r):
                 sess_name, src, dest, port)
 
     print("\n****************\n{}; \n****************".format(info_str))
```

Comment:
Is this ***really*** necessary and important to emphasize with this many asterisks?
I'd prefer to use a verbosity flag to print extra information and keep the command's output minimal, for the sake of scripting support.